### PR TITLE
chore: set explicit timeouts on webhooks

### PR DESCRIPTION
**Pull request by ccojocar**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

It sets explicit timeouts shorter than the default. This is required by our webhook configuration static check which doesn't allowed longer timeouts than 5 seconds.

#### Does this PR introduce a user-facing change?

```release-note
NONE

```

### CODE REVIEW

File: deploy/overlays/webhook/webhook_config.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
 webhooks:
   - name: binding.spo.io
     failurePolicy: Fail
+    timeoutSeconds: 5
```
Comment (jhrozek) [inline deploy/overlays/webhook/webhook_config.yaml:12]: If this is realistically something that deployers want configurable, why not make it configurable? (What if your policy will say no longer than 4 seconds next year)?

Comment (ccojocar) [reply deploy/overlays/webhook/webhook_config.yaml:12]: Happy to make it configurable. How do you suggest to do that in this situation?

Comment (jhrozek) [reply deploy/overlays/webhook/webhook_config.yaml:12]: ugh, I'm sorry, only now did I notice that this is the overlay, for some reason I thought you were modifying the go sources in bindata with a hardcoded config. Sorry, going to ack this now.

**Review by jhrozek — APPROVED** [review]
/lgtm

File: deploy/overlays/webhook/webhook_config.yaml (hunk 1 above)
Comment (ccojocar) [reply deploy/overlays/webhook/webhook_config.yaml:12]: Thanks for reviewing it!

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**